src/collider/README.md

```diff
@@ -38,3 +37,49 @@ A websocket-based signaling server in Go.
 
     go test collider
 
+## Deployment
+
+Change [roomSrv](https://github.com/webrtc/apprtc/blob/master/src/collider/collidermain/main.go#L16) to your AppRTC server instance e.g.
```

Comment:
It's probably a good idea to mention what kind of system these instructions were created from, since I assume log rotation and even inittab may be different on different Linux distros.